types/src/validator_config.rs

```diff
@@ -13,7 +13,6 @@ use serde::{Deserialize, Serialize};
 pub struct ValidatorConfigResource {
     pub validator_config: Option<ValidatorConfig>,
     pub delegated_account: Option<AccountAddress>,
-    pub is_certified: bool,
```

Comment:
oh wow, so we're getting rid of all this too :/

I guess simplicity is best...

Reply:
We had the is_certified flag before in order to give the Association a way to remove validator from the set - it could set the is_certified to false. Now, the add/remove validator are callable by the Association only, so it seems there is no good reason to keep this flag. For additional certification/decertification the freeze/unfreeze methods on the account could be used.

But if you think that this flag is still useful, we can keep it - please let me know!